pkg/manifest/manifest.go

```diff
@@ -514,6 +515,10 @@ func UnmarshalManifest(manifestData []byte) (*Manifest, error) {
 		if _, found := knownFields[key]; found {
 			delete(unmappedData, key)
 		}
+		// Explicitly error out if deprecated invocationImage is supplied
+		if key == "invocationImage" {
```

Comment:
I think this would be a rough transition for some people who have older manifests that set the invocationImage. What do you think about ignoring it and printing this message as a warning instead? We'd have to pass in the porter context to this function.

Otherwise someone updating to this release would suddenly have porter build completely stop working. Then perhaps in a later release we can make it a hard error, perhaps in 1.0 we can hard fail on unknown keys.

What do you think?

Reply:
Agreed, I'll update to print a warning instead.

Reply:
Updated in https://github.com/deislabs/porter/pull/1140/commits/abb969a4c9e0feb196e8b0682f9bc7f3e69cd6f4 - let me know how it looks when convenient.